### ISSUE TITLE
Add RSS for news

### ISSUE THREAD

**Issue by kuchta** [freenet/freenetorg-website#10]
Please add RSS for news so that we can stay in touch :)

**Comment by sanity** [comment]
Does this work for you? https://nitter.net/FreenetOrg

**Comment by kuchta** [comment]
Yep, that works. Thank you 🙏🏻

Maybe that feed could be linked to website so that RSS readers could find it automatically. Like: https://www.petefreitag.com/item/384.cfm

**Comment by sanity** [comment]
Yes, I'll think about the best way to do it. Ideally, the news section of the website would be linked to the twitter feed.
